docs/index.md

```diff
@@ -18,7 +18,6 @@
 * [Chat API](chat.md)
 * [Webinar API](webinar.md)
 * [Internal Signaling API](internal-signaling.md)
-* [Standalone Signaling API](standalone-signaling-api-v1.md)
```

Comment:
Would something like a link to `https://nextcloud-spreed-signaling.readthedocs.io/en/latest/` work?
Otherwise I would keep the entry and replace the content of the `standalone-signaling-api-v1.md` with a link to https://nextcloud-spreed-signaling.readthedocs.io/en/latest/

Reply:
Good point, changed and squashed.